src/libraries/Microsoft.AspNetCore.Internal.Transport/src/Microsoft.AspNetCore.Internal.Transport.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <!-- These are wrapper project files for packaging.-->
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.0</TargetFrameworks>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
You can remove this line as well by updating to a newer NoTargets SDK version: https://github.com/microsoft/MSBuildSdks/blob/1a1e144dd2d5116a280b3f87e1d4ff49c30619b4/src/NoTargets/Sdk/Sdk.props#L34.